src/resources/FundraisingTeams.js

```diff
@@ -9,7 +9,7 @@ class FundraisingTeams extends Resource {
         'fundraising-pages',
         'posts'
       ],
-      creates: ['posts'],
+      creates: ['posts', 'fundraising-pages'],
```

Comment:
This one's interesting. Since a POST to /fundraising-teams/#/fundraising-pages doesn't actually create the page, we may want to call this something other than `classy.fundraisingTeams.createFundraisingPage()`. Maybe `classy.fundraisingTeams.addFundraisingPage()`?

If it is creating and associating the page at the same time,